configstack/test_helpers.go

```
@@ -157,7 +157,7 @@ func state(t *testing.T, bucket string, key string) *remote.RemoteState {
 // Create a mock TerragruntOptions object and configure its RunTerragrunt command to return the given error object. If
 // the RunTerragrunt command is called, this method will also set the executed boolean to true.
 func optionsWithMockTerragruntCommand(terragruntConfigPath string, toReturnFromTerragruntCommand error, executed *bool) *options.TerragruntOptions {
-	opts := options.NewTerragruntOptionsForTest(terragruntConfigPath)
+	opts, _ := options.NewTerragruntOptionsForTest(terragruntConfigPath)
```

Same thing here. Don't ignore errors. It always comes back to haunt you!